config/service.go

```diff
@@ -160,7 +160,7 @@ func (c *ServiceConfig) Finalize() {
 		c.Tag = String("")
 	} else {
 		log.Println("[WARN] (config) The 'tag' attribute was marked for " +
-			"deprecation in v0.2.0 and will be removed in a future version " +
+			"deprecation in v0.2.0 and will be removed in v0.4.0 " +
```

Comment:
If you weren't already planning on it, adding this to the CHANGELOG line would be useful too

Reply:
Will do!